src/main/webapp/WEB-INF/themes/desktop/ftl/article/aside/relatedArticles.ftl

```diff
@@ -20,32 +20,38 @@
   ~ DEALINGS IN THE SOFTWARE.
   -->
 
-
-<#if relatedArticles?size gt 0>
-
-<div class="related-articles-container">
-  <h3><#include "relatedArticleTitle.ftl"/></h3>
-  <ul class="related-articles">
-  <#list relatedArticles as relatedArticle>
-      <li>
-        <@xform xml=relatedArticle.title/>
-        <ul class="related-article-links">
-          <li class="related-article-link-page">
-            <a href="<@siteLink handlerName="article" journalKey=relatedArticle.journal.journalKey
-                queryParameters={"id": relatedArticle.doi} />">View Page</a>
-          </li>
-          <!--  Annotations and issue images do not have printables. They can be filtered by doi. -->
-          <#if relatedArticle.doi?contains("10.1371/journal")>
-            <li class="related-article-link-download">
-              <a href="<@siteLink handlerName="assetFile" journalKey=relatedArticle.journal.journalKey
-                  queryParameters={"type": "printable", "id": relatedArticle.doi} />"
-                  target="_blank" title="PDF opens in new window">PDF</a>
-          </li>
-        </#if>
-        </ul>
-      </li>
-  </#list>
-  </ul>
-</div>
-
+<#if relatedArticlesByType?keys?size gt 0>
+  <div class="related-articles-container">
+    <h3><#include "relatedArticleTitle.ftl"/></h3>
+    <ul class="related-articles">
+      <#list relatedArticlesByType as type, articles>
+        <li>
+          <b>
+            <#if articles[0].hasRelation>has</#if>
+            <@pluralize count=articles?size value=type?upper_case />
+            <#if articles[0].pertainsToRelation>pertains to</#if>
+          </b>
+        </li>
+          <#list articles as article>
+            <li>
+              <@xform xml=article.title/>
+              <ul class="related-article-links">
+                <li class="related-article-link-page">
+                  <a href="<@siteLink handlerName="article" journalKey=article.journal.journalKey
+                                      queryParameters={"id": article.doi} />">View Page</a>
+                </li>
+                <!--  Annotations and issue images do not have printables. They can be filtered by doi. -->
+                <#if article.doi?contains("10.1371/journal")>
```

Comment:
With these changes rhino could include a more definitive `related_article.printable` method instead of the quick solution I baked up here.  Not that we have to do it now, but it is facilitated by this PR.